backend/docs/development_docs/api_requests_per_screen.rst

```diff
@@ -0,0 +1,272 @@
+Mapping of backend requests (SOLR queries) to application screens
+=================================================================
+
+This document describes how to get the data necessary for displaying the
+various pages in the phone book mockup. The document will refer to the
+mockups for the web version. These mockups may be found in the same
+directory as the present document, in the file
+``OS2-telefonbog_webversion.pdf``.
+
+The document follows the mockups, describing the necessary queries page
+by page. 
+
+In these examples, the backend (an SOLR server) is assumed to be running
+on localhost (``127.0.0.1``) and port 8983. In production, this will be
+different, and the URLs should be adjusted accordingly.
+
+**Note:** For more advanced search combinations than described in this
+document, there's this quick tutorial:
+
+    http://www.solrtutorial.com/solr-query-syntax.html
+
+For more information, please consult the SOLR reference manual:
+
+    http://lucene.apache.org/solr/guide/7_5/overview-of-searching-in-solr.html#overview-of-searching-in-solr
+
+It is possible to combine query parameters in many ways not described
+above.
+
+Here, only the SOLR search syntax specifically needed to reproduce the
+searches in the mockups is given. Suffice it to say that the full
+SOLR/Lucene search syntax has many more features than covered here.
+
+Front page
+==========
+
+Root organisation
+-----------------
+
+To get the name of the root organisation ("Ballerup Kommune" in the
+example), you have two options.
+
+In the first case, you rely on the assumption that this organisation has
+one root organisation unit only. In that case, you can search for the
+single root node, denoted by having its "parent" field set to the value
+"ROOT": ::
+
+   http://127.0.0.1:8983/solr/departments/select?q=parent:ROOT
+
+If you cannot make that assumption, e.g. because there are more than one
```

Comment:
We expect the common case to be more than one root unit — so I'd switch these around and describe the case of more than one root first.

Unless, of course, your importer allows restricting the import to a subtree — which might make sense.